webui/ocw/lib/EC2.py

```diff
@@ -67,3 +74,51 @@ def list_regions(self):
 
     def delete_instance(self, instance_id):
         self.ec2_resource().instances.filter(InstanceIds=[instance_id]).terminate()
+
+    def cleanup_all(self):
+        response = self.ec2_client().describe_images(Owners=['self'])
+        images = dict()
+        for img in response['Images']:
+            # 'CreationDate': '2019-10-22T20:40:45.000Z',
+            # 'ImageId': 'ami-00d30c03d17d3db69',
+            # 'Name': 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-BYOS-Build1.55.raw.xz'
+            logger.debug("[EC2] Found image '{}'".format(img['Name']))
```

Comment:
`[EC2]` - isn't we can get same info from logger name ?

Reply:
Yes, this is how a log look like:
```
2019-11-11 10:22:48,185 ocw.lib.EC2  DEBUG    [EC2] Found image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-On-Demand-Build1.58.raw.xz'
2019-11-11 10:22:48,189 ocw.lib.EC2  DEBUG    [EC2] Found image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-BYOS-Build1.58.raw.xz'
2019-11-11 10:22:48,190 ocw.lib.EC2  DEBUG    [EC2] Found image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-BYOS-Build1.56.raw.xz'
2019-11-11 10:22:48,190 ocw.lib.EC2  DEBUG    [EC2] Found image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-On-Demand-Build1.56.raw.xz'
2019-11-11 10:22:48,191 ocw.lib.EC2  INFO     [EC2] Delete image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-On-Demand-Build1.56.raw.xz' (ami:ami-0e0f2XXXX77a06b09)
2019-11-11 10:22:48,288 ocw.lib.EC2  INFO     [EC2] Delete image 'openqa-SLES12-SP5-EC2.x86_64-0.9.1-BYOS-Build1.56.raw.xz' (ami:ami-0b39dXXXXd8fd5ad)
```

Reply:
ok , let's leave it for now . I think it would be more clear to come up with separate PR for this ...